graphene/core/schema.py

```diff
@@ -69,8 +67,7 @@ def T(self, _type):
     @property
     def executor(self):
         if not self._executor:
-            self._executor = Executor(
-                [SynchronousExecutionMiddleware()], map_type=OrderedDict)
+            self._executor = SyncExecutor()
```

Comment:
Ah, got it. I removed that logic - still having trouble with the tests, though. I'll give it another shot later today